apps/web/app/(authenticated)/(app)/app/apis/client.tsx

```diff
@@ -55,8 +55,8 @@ export function ApiList({ apis }: { apis: ApiWithKeys }) {
             <Link key={api.id} href={`/app/apis/${api.id}`}>
               <Card className="hover:border-primary/50 group relative overflow-hidden duration-500 ">
                 <CardHeader>
-                  <div className="flex items-center justify-between ">
-                    <CardTitle className="overflow-hidden text-ellipsis whitespace-nowrap">
+                  <div className="flex items-center justify-between">
+                    <CardTitle className="overflow-hidden text-ellipsis whitespace-nowrap pb-2">
```

Comment:
I think this is more of an underlying issue and we should find a solution in the actual <Card> component rather than doing a manual fix here

Reply:
Would you suggest modifying the <CardTitle> component?